### PR TITLE
Provide a template for new PRs

### PR DESCRIPTION
**Pull request by olethanh**
As discussed, providing these informations on each PR should improve review and code quality

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- the behaviour to expect, e.g: tooltip has purple background color because the client likes it so, changed a key in the API response to be consistent with other endpoints
+
+## How to test
+
+Explain how to test your PR.
+If a specific config is required explain it here (account, data entry, ...)
+
+## Print screen / video
+
+Upload here print screens or videos showing the changes if relevant.
```
Comment (Psycojoker) [inline .github/PULL_REQUEST_TEMPLATE.md:28]:
```suggestion
Upload here screenshots or videos showing the changes if relevant.
```

**Review by Psycojoker — APPROVED** [review]
Looks very good to me :+1:

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Related ClickUp, GitHub or Jira tickets : ALEPH-XXX
+
+## Self proofreading checklist
+
+- [ ] Is my code clear enough and well documented
+- [ ] Are my files well typed
+- [ ] New translations have been added or updated if new strings have been introduced in the frontend
+- [ ] Database migrations file are included
+- [ ] Are there enough tests
+- [ ] Documentation has been included (for new feature)
```
Comment (hoh) [inline .github/PULL_REQUEST_TEMPLATE.md:12]:
```suggestion
- [ ] The new code clear, easy to read and well commented.
- [ ] New code does not duplicate the functions of builtin or popular libraries.
- [ ] An LLM was used to review the new code and look for simplifications.
- [ ] New classes and functions contain docstrings explaining what they provide.
- [ ] All new code is covered by relevant tests.
- [ ] Documentation has been updated regarding these changes.
```

**Comment by hoh** [comment]
@olethanh  Do we want to add this template to other projects ?

**Pull request merged**